src/components/Loader/Loader.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {Story} from '@storybook/react'
+import {Loader} from './index';
+import {LoaderProps} from './Loader.types';
+
+export default {
+    title: 'Components/Loading',
+    component: Loader
+};
+
+const Template: Story<LoaderProps> = args => <Loader {...args}/>;
+
+export const Loading = Template.bind({});
+Loading.args = {
```

Comment:
If you install `@storybook/addon-controls` and add it to the list of addons in main.js, then the controls should show up in the story.

Reply:
Can we have `addon-knobs` and `addon-controls` to the same project ? Or do we prefer to migrate all stories to `addon-essentials` at the same time ?

Reply:
I think it could be good to slowly start migrating to `addon-controls` when we can and turn `addon-knobs` off for those components which have controls.

Later, when everything has been changed to controls, we can remove `@storybook/addon-knobs`, `@storybook/addon-controls`, and add `@storybook/addon-essentials`. Sound okay?

Reply:
I added `@storybook/addon-controls`